TypeCobol/Compiler/Scanner/Scanner.cs

```diff
@@ -236,17 +236,11 @@ private static bool IsDebuggingModeActive(MultilineScanState lineScanState, stri
                  * So an inactive debug line has to be parsed as regular source if it participates
                  * in a REPLACE directive.
                  */
-                return lineScanState.InsideReplaceDirective || StartsWithReplace();
+                return lineScanState.InsideReplaceDirective || lineSourceText.StartsWith(nameof(TokenType.REPLACE), StringComparison.OrdinalIgnoreCase);
```

Comment:
It works because REPLACE token doesn't contains an `_` but I'd rather keep the original code in order to cover further cases with token that contains a `_`.

Reply:
I've restored the original code but this forces us to check for non-null of the `string replaceKeyword` which is a bit odd since we know at compile-time that this is always true. That's why i went for the `nameof` approach instead.

Reply:
null-forgive on result